docs/application/dotnet/guides/nui/text.md

```diff
@@ -396,12 +412,28 @@ field.PlaceholderTextFocused = "Enter Name";
 window.Add(field);
 ```
 
-When the `TextField` is tapped, it automatically gets the keyboard focus. Key events correspond to entering the text. Additionally, the placeholder text is removed as soon as the text is entered. The text entered can be retrieved by using the `TEXT` property:
+When the `TextField` is tapped, it automatically gets the keyboard focus. Key events correspond to entering the text. Additionally, the placeholder text is removed as soon as the text is entered. The text entered can be retrieved by using the `Text` property:
 
 ```csharp
 string fieldTextString = field.Text;
 ```
 
+### Use Placeholder Property in TextField

```

Comment:
Global: 
Can we use the Sentence case for all headings? Let me know if that is possible.

Reply:
I am sorry, what is Sentence case?

Reply:
Class or property names cannot be written in lowercase.
In other cases I can change it.

For example :
### Use Placeholder Property in TextField -> ### Use Placeholder property in TextField.
(Placeholder = property, TextField = class)

### Align Text in TextField -> ### Align text in TextField
### Use HiddenInputSettings Property in TextField -> ### Use HiddenInputSettings property in TextField

Please let me know if you want this change for all headings.

Reply:
Oh sorry! I didn't mean that exactly.

What my intention was that heading of this page which starts with (#, ##, ###) are, written in Title case (we capitalize ist letter of each word in a title or heading)

I suggested using sentence case for headings, which means:

In sentence case, lowercase most words in a title or heading. Capitalize only the Ist letter of the sentence: 


e.g 

### Use Placeholder Property  (Title case)

### Use placeholder property (Sentence case) 

If there is a name in the sentence case, we write that as capital.

### Use placeholder property in TextField (correct)
### Use placeholder property in textfield (Incorrect)

Reply:
Thanks for your explanation.

By the way, in below case,
### Use placeholder property in TextField (correct)


"Placeholder" is the name of the property(attribute).
So, I think below is correct.
What do you think?
### Use Placeholder property in TextField